platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultFileLocation.java

```diff
@@ -45,20 +39,22 @@ public String getType() {
         return "file";
     }
 
+    @Override
     public String getPath() {
         return path;
     }
-    @Nullable
+
+    @Override
```

Comment:
❔ is this implicitely nullable because the constructor parameter is?

Reply:
Probably. It doesn't matter really as I'm working hard to refactor that part of the API separately.